tests/features/payment-gateway-suggestions/payment-gateway-suggestions.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * Test the class that parses the payment suggestions.
+ *
+ * @package WooCommerce\Admin\Tests\PaymentGatewaySuggestions
+ */
+
+use Automattic\WooCommerce\Admin\Features\PaymentGatewaySuggestions\Init as PaymentGatewaySuggestions;
+
+/**
+ * class WC_Tests_PaymentGatewaySuggestions_Init
+ */
+class WC_Tests_PaymentGatewaySuggestions_Init extends WC_Unit_Test_Case {
+
+	/**
+	 * Set up.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		add_filter(
+			'transient_' . PaymentGatewaySuggestions::SPECS_TRANSIENT_NAME,
+			function( $value ) {
+				if ( $value ) {
+					return $value;
+				}
+
+				return array(
+					array(
+						'id' => 'default-gateway',
+					),
+				);
+			}
+		);
+	}
+
+	/**
+	 * Tear down.
+	 */
+	public function tearDown() {
+		parent::tearDown();
+		PaymentGatewaySuggestions::delete_specs_transient();
+		remove_all_filters( 'transient_' . PaymentGatewaySuggestions::SPECS_TRANSIENT_NAME );
+	}
+
+	/**
+	 * Add test specs.
+	 */
+	public function get_mock_specs() {
+		return array(
+			array(
+				'id'         => 'mock-gateway',
+				'is_visible' => (object) array(
+					'type'      => 'base_location_country',
+					'value'     => 'ZA',
+					'operation' => '=',
+				),
+			),
+		);
+	}
+
+	/**
+	 * Test that default gateways are provided when remote sources don't exist.
+	 */
+	public function test_get_default_specs() {
+		add_filter(
+			'woocommerce_admin_payment_gateway_suggestions_data_sources',
+			function() {
+				return array();
+			},
+			PHP_INT_MAX
+		);
+		$specs = PaymentGatewaySuggestions::get_specs();
+		$this->assertTrue( ! empty( $specs ) );
```

Comment:
Could we make this one a little more specific then just not empty, maybe comparing directly with `DefaultPaymentGateways::get_all()`?

Reply:
That's a much better assertion- thanks!